eng/common/cross/build-rootfs.sh

```diff
@@ -424,6 +452,11 @@ case "$__AlpineVersion" in
         elif [[ "$__AlpineArch" == "riscv64" ]]; then
             __AlpineLlvmLibsLookup=1
             __AlpineVersion=edge # minimum version with APKINDEX.tar.gz (packages archive)
+        elif [[ "$__AlpineArch" == "loongarch64" ]]; then
+            __AlpineLlvmLibsLookup=1
+            if [[ "$__AlpineMajorVersion" -lt 3 || "$__AlpineMinorVersion" -lt 21 ]]; then
+                __AlpineVersion=3.21 # minimum version with loongarch64 support
+            fi
```

Comment:
I'd merge these two branches and set them to alpine 3.21 and llvm 19:

```suggestion
        elif [[ "$__AlpineArch" == "loongarch64" || "$__AlpineArch" == "riscv64" ]]; then
            __AlpineVersion=3.21
            __AlpinePackages+=" llvm19-libs"
```